container_files/tutorials/Loading data from HTTP servers or online file storage services.ipynb

```diff
@@ -0,0 +1,156 @@
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Loading data from an HTTP server\n",
+    "\n",
+    "MLDB gives users full control over where and how data is persisted. MLDB handles multiple protocol for URLs (see [Files and URLs](../../../../doc/#builtin/Url.md.html)). In this tutorial, we provide examples to load files via:\n",
+    "* <code> file:// </code> for a file inside an MLDB container (see the [Loading Data Tutorial](../../../../ipy/notebooks/_tutorials/_latest/Loading%20Data%20Tutorial.ipynb) for an example)\n",
+    "* <code> http:// </code> or <code> https://</code> for a file accessible on a HTTP server on the public internet or a private intranet "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The notebook cells below use `pymldb`'s `Connection` class to make [REST API](../../../../doc/#builtin/WorkingWithRest.md.html) calls. You can check out the [Using `pymldb` Tutorial](../../../../doc/nblink.html#_tutorials/Using pymldb Tutorial) for more details."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "from pymldb import Connection\n",
+    "mldb = Connection()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Loading data with http:// or https://\n",
+    "MLDB makes it very easy to load with the public web. A powerful feature of MLDB allows us to load publicly avaialbe data directly from a web address. The datasets below are from the [Stanford Network Analysis Project (SNAP)](http://snap.stanford.edu/index.html). Here, we have chosen to load [Facebook's Social Circles](http://snap.stanford.edu/data/egonets-Facebook.html) dataset. In our first example, we will be [`importing a text file`](../../../../doc/#builtin/procedures/importtextprocedure.md.html)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<Response [201]>\n"
+     ]
+    }
+   ],
+   "source": [
+    "dataUrl = \"http://snap.stanford.edu/data/facebook_combined.txt.gz\"\n",
+    "\n",
+    "print mldb.put(\"/v1/procedures/import_data\", {\n",
+    "    \"type\": \"import.text\",\n",
+    "    \"params\": {\n",
+    "        \"dataFileUrl\": dataUrl,\n",
+    "        \"headers\": [\"node\", \"edge\"],\n",
+    "        \"delimiter\": \" \", \n",
+    "        \"quotechar\": \"\",\n",
+    "        \"outputDataset\": \"import_URL1\",\n",
+    "        \"runOnCreation\": True\n",
+    "    }\n",
+    "})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If the targeted file is inside an archive, we can specify the specific path to the file, as seen in the example below. Here, we load the '3980.circles' file within the 'facebook' folder."
```

Comment:
Ohh ok donc j'avais pas encore lu ceci (par rapport au commentaire précédent). Je combinerais mon truc plus haut avec ton commentaire. We can decrompress transparently. Mais si le fichier compressé peut contenir plus d'un fichier à l'intérieur (.tar ou .zip), on peut accéder à fichier spécifique comme ceci genre